### ISSUE TITLE
Existing comment element in core system.xml file can not be overwritten

### ISSUE THREAD

**Issue by JanuszWitrykus** [magento/magento2#21027]
### Preconditions (*)

1. Magento 2 version 2.2.7

### Steps to reproduce (*)

In custom module I try to overwrite comment field from core module in which comment already exists, like this (in system.xml, module has set correct sequence):
`
...
\<system\>
\<section id="currency"\>
\<group id="options"\>
\<field id="base" translate="label comment" type="select" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="0" canRestore="1"\>
                    \<comment\>
                        \<![CDATA[Base market currency]]\>
                    \</comment\>
...

`

Core module:
Magento_Directory, version 100.2.6

### Expected result (*)

Comment field should be overwritten and visible in admin as "Base market currency"

### Actual result (*)

Comment field is not overwritten and visible in admin as "Base currency is used for all online payment transactions. The base currency scope is defined by the catalog price scope ("Catalog" > "Price" > "Catalog Price Scope")."

**Comment by magento-engcom-team** [comment]
Hi @JanuszWitrykus. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@JanuszWitrykus do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
HI @JanuszWitrykus thank you for you report, i'm not able to reproduce following steps you described, please set the following -> `Vendor/ModuleName/etc/module.xml`

```
<?xml version="1.0"?>
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
<module name="Vendor_YourModule" setup_version="1.0.0">
<sequence>
<module name="Core_ModuleName"/>
</sequence>
</module>
</config>
```

**Comment by JanuszWitrykus** [comment]
Hi @engcom-backlog-nazar  , thank you very much for you effort. I have correct module.xml. And I can overwrite label element (always) but comment element only if it is empty in the core module.

**Comment by JanuszWitrykus** [comment]
Hi @magento-engcom-team @engcom-backlog-nazar  can you please reopen this issue? I have tested it on several Magento instances, this is simple to reproduce.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98170`, `MAGETWO-98171` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by Renu05** [comment]
Hi @JanuszWitrykus ,

I have created a custom module to change the system.xml comment text and the text got successfully updated in 
Stores > Configuration > Catalog > Catalog > Price"> Catalog Price Scope

**This has been tested in Magento 2.3.3 CE version**

1. **Create module.xml** :
```
<module name="Vendor_ModuleName">
        <sequence>
            <module name="Magento_Catalog"/>
        </sequence>
 </module>
```

2. Create a system.xml to override the comment.
```
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
    <system>
        <section id="catalog" translate="label" sortOrder="60" showInDefault="1" showInWebsite="1" showInStore="1">
            <group id="price" translate="label" type="text" sortOrder="400" showInDefault="1" showInWebsite="0" showInStore="0">
                <label>Price</label>
                <field id="scope" translate="label comment" type="select" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="0">
                    <comment><![CDATA[Base market currency.]]></comment>
                </field>
            </group>
        </section>
    </system>
</config>
```


Hence this is not an issue so closing the ticket.

**Comment by JanuszWitrykus** [comment]
@Renu05 Thank you very much for your effort, but can you tell me please, why did you close this bug? In Preconditions I said, that this bug exists in Magento 2.2.7. Have we any patch to fix this bug in 2.2.7? Please, don't test in in 2.3.x, this report does not apply to the version 2.3.x. I hope, that M 2.2.7 is still supported :)

@magento-engcom-team  Can you please confirm, that we have patch to fix this bug in 2.2.x? If not, maybe it can be still open to wait on solution?
